src/Symfony/Component/HttpKernel/Fragment/InlineFragmentRenderer.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\HttpKernel\Fragment;
 
+use Symfony\Component\Debug\Exception\FatalThrowableError;
```

Comment:
For some reason, the build failed because this class could not be autoloaded in `HttpKernel`. So I've made another copy :-/